packages/core/e2e/examples/nextjs-agent/next.config.mjs

```diff
@@ -1,5 +1,14 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
+const nextConfig = {
+  webpack: (config) => {
+    config.resolve.alias = {
+      ...config.resolve.alias,
+      "@smithy": false,
+      "@aws-sdk": false,
+    };
+    return config;
+  },
```

Comment:
should not modify this in user side

Reply:
@himself65 I wasn't sure where it should be modified, it seems to be specific only to this example! `packages/core/src/next.ts` didn't seem like the right place, where shall I place it?

Reply:
Yes